modules/master-node-sizing.adoc

```diff
@@ -33,24 +33,26 @@ The control plane node resource requirements depend on the number and type of no
 
 | 252
 | 4000
-| 16
-| 64
+| 24
+| 64, but 128 if using `OVNKubernetes`
 
 | 501
 | 4000
 | 16
-| 96
+| 96, but untested with `OVNKubernetes`
```

Comment:
I think is better to move the OVN statement to column with the number of worker nodes as I suggested